ui/appui-react/src/appui-react/icons/AssetClassificationHide.tsx

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module AssetClassificationHide
```

Comment:
You can find available modules in `appui-react.ts`
These are used to build the docs site https://www.itwinjs.org/reference/appui-react/utilities/
Since the APIs are `@internal` they will not be exposed in the doc site.

```suggestion
 * @module Utilities
```

Reply:
Done